plugin/trino-druid/src/test/java/io/trino/plugin/druid/BaseDruidConnectorTest.java

```diff
@@ -369,4 +369,12 @@ public void testLimitPushDown()
                         "LIMIT 30"))
                 .isNotFullyPushedDown(joinOverTableScans);
     }
+
+    @Test
+    public void testPredicatePushdown()
+    {
+        assertThat(query("SELECT * FROM orders where __time > DATE'1970-01-01'")).isFullyPushedDown();
```

Comment:
Copy the tests from TestPostgreSqlConnectorTest#testPredicatePushdown and remove the ones not relevant for Druid. Keeping tests consistent is useful and allows extracting them to base test classes in the future.

For `__time` pushdown add a separate test method since that's something specific to Druid.